src/services/llms/base.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from typing import List, Dict, Any
+from ...core.models import PRDetails
+
+
+class BaseLLMService(ABC):
+    @abstractmethod
+    def create_prompt(self, file_path: str, diff_content: str, pr_details: PRDetails) -> str:
+        pass
+    
+    @abstractmethod
```

Comment:
The return type of `get_review` is `List[Dict[str, Any]]`. This is very generic.  Can this be made more specific? Defining a custom data structure for the review would make the code more robust, readable and maintainable.  For example, a `Review` class could be defined to represent the structure of the review data more precisely.